doc/users/next_whats_new/legend_shadow_colors.rst

```diff
@@ -0,0 +1,5 @@
+Configurable legend shadows
+-------------------------
+The *shadow* parameter of legends now accepts dicts in addition to booleans.
+Dictionaries can contain any keywords for `.patches.Patch`.
```

Comment:
This sounds very generic. I think the practical parameters are very limited; maybe even only `color` and `alpha`? The latter is not even necessary since we recently added the possibility to specify any color as a tuple `(color, alpha)`.

We should mention the really practical parameters.

--

Semi-OT: What would be useful is supporting the offesets `ox`, `oy` passed to `Shadow`. These are currently hard-coded. - Not necessarily part of this PR, but a bit of justification that a parameter dict is better than just a color name.

Reply:
`ox` and `oy` are now supported. I am not sure what other arguments are useful either, so I expanded with two specific examples. What do you think?